data-processing-lib/ray/src/data_processing/transform/table_transform.py

```diff
@@ -59,17 +59,19 @@ def transform_binary(self, byte_array: bytes, ext: str) -> tuple[list[tuple[byte
         # Add number of rows to stats
         stats = stats | {"source_doc_count": table.num_rows}
         # convert tables to files
-        return self._check_and_convert_tables(out_tables=out_tables, stats=stats | {"source_doc_count": table.num_rows})
+        return self._check_and_convert_tables(
+            out_tables=out_tables, stats=stats | {"source_doc_count": table.num_rows}
+        )
 
-    def transform(self, table: pa.Table) -> tuple[list[pa.Table], dict[str, Any]]:
-        """
-        Converts input table into an output table.
-        If there is an error, an exception must be raised - exit()ing is not generally allowed when running in Ray.
-        :param table: input table
-        :return: a tuple of a list of 0 or more converted tables and a dictionary of statistics that will be
-        propagated to metadata
-        """
-        raise NotImplemented()
+    # def transform(self, table: pa.Table) -> tuple[list[pa.Table], dict[str, Any]]:
+    #     """
+    #     Converts input table into an output table.
+    #     If there is an error, an exception must be raised - exit()ing is not generally allowed when running in Ray.
+    #     :param table: input table
+    #     :return: a tuple of a list of 0 or more converted tables and a dictionary of statistics that will be
+    #     propagated to metadata
+    #     """
+    #     raise NotImplemented()
 
```

Comment:
We're using the generic defintion of transform() from the super class so it is not needed here.

Reply:
Again, we sacrifice the readability of the code, and what is the gain? A simple cost/benefit analysis

Reply:
Copying from below
`This comes in because of the AbstractTransformLauncher -> runtime config -> transform config -> AbstractTransform. Spark code is using the AbstractTransformLauncher and therefore we have a common AbstractTransform across all runtimes/launchers.`

Reply:
We have a common/unified framework, not independent ones.